lte/gateway/c/oai/tasks/amf/Registration.cpp

```diff
@@ -800,33 +785,25 @@ int amf_proc_registration_complete(
        * ACCEPT message as valid.
        */
       amf_ctx_set_attribute_valid(amf_ctx, AMF_CTXT_MEMBER_GUTI);
-    } else {
-      OAILOG_DEBUG(
-          LOG_NAS_AMF,
-          "UE " AMF_UE_NGAP_ID_FMT
-          " REGISTRATION COMPLETE discarded (AMF procedure not found)\n",
-          ue_id);
-      bdestroy((bstring)(smf_msg_pP));
     }
   } else {
-    OAILOG_WARNING(LOG_NAS_AMF, "UE Context not found..\n");
+    OAILOG_WARNING(LOG_NAS_AMF, "UE Context not found...ue_id=%d\n", ue_id);
```

Comment:
utilize AMF_UE_NGAP_ID_FMT rather than `%d`.

Reply:
Comments addressed. Fixed same in other parts of PR as well.